### PR TITLE
Contributor docs: Minor edits for consistency

### PR DESCRIPTION
**Pull request by dipikabh**
MDN > Community> Users and teams

### CODE REVIEW

File: files/en-us/mdn/community/users_teams/index.md [hunk 1]
```diff
@@ -9,103 +9,105 @@ tags:
...
-- **Yari content ZH**: Responsible for reviewing `zh` docs on MDN
+
+  The following teams are specific to respective localization teams:
+  - **Yari content ES**: Responsible for reviewing MDN docs in spanish
+  - **Yari content FR**: Responsible for reviewing MDN docs in french
+  - **Yari content JA**: Responsible for reviewing MDN docs in japanese
+  - **Yari content KO**: Responsible for reviewing MDN docs in korean
+  - **Yari content PT-BR**: Responsible for reviewing MDN docs in brazilian portuguese
+  - **Yari content RU**: Responsible for reviewing MDN docs in russian
+  - **Yari content ZH**: Responsible for reviewing MDN docs in chinese
```
Comment (schalkneethling) [inline files/en-us/mdn/community/users_teams/index.md:43]: Should the names of the languages not be capitalised? ie. Spanish not spanish?

**Comment by dipikabh** [comment]
To be pedantic, should this page have the title "Teams and owners" instead of "Users and teams"?

File: files/en-us/mdn/community/users_teams/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md:43]: Yes, they should be. Thanks for catching @schalkneethling! I'll fix this.

**Comment by Rumyra** [comment]
I'm happy with the suggeston @dipikabh - maybe 'Owners and teams' (?)

**Review by schalkneethling — APPROVED** [review]
Thank you, @dipikabh 🎉

**Pull request merged**